poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```

Comment:
Can you inform the rest of the developers about upgrading Poetry? Sometimes they generate diffferent lock files. Also update poetry-version other places where we use it?